packages/integrations/cloudflare/src/index.ts

```diff
@@ -238,11 +243,16 @@ export default function createIntegration(args?: Options): AstroIntegration {
 						hybridOutput: 'stable',
 						staticOutput: 'unsupported',
 						i18nDomains: 'experimental',
-						sharpImageService: {
-							support: 'limited',
-							message:
-								'Cloudflare does not support sharp. You can use the `compile` image service to compile images at build time. It will not work for any on-demand rendered images.',
-						},
+						// For explicitly set image services, this hack is used to suppress the warning about using `sharp`
+						// by inferring the user is aware of its implications.
+						// TODO: If an option to supress the warnings for `supportedAstroFeatures` is added, we should use it instead.
+						sharpImageService: args?.imageService
+							? 'stable'
+							: {
+									support: 'limited',
+									message:
+										'Cloudflare does not support sharp at runtime, but the `compile` image service can be used to optimize images with sharp on pre-rendered pages during build time.',
```

Comment:
```suggestion
										'Cloudflare does not support sharp at runtime. However, you can configure `image service:  'compile'` to optimize images with sharp on prerendered pages during build time.',
```
Felt like a bit of a mouthful all in one sentence. What about something like this, Yan?